packages/onchainkit/src/fund/types.ts

```diff
@@ -106,20 +102,27 @@ export type FundButtonRenderParams = {
   isDisabled: boolean;
 };
 
+// Require at least one funding source for FundButton
+type FundButtonSourceProps =
+  | { fundingUrl: string; sessionToken?: string }
+  | { sessionToken: string; fundingUrl?: string };
```

Comment:
Do we allow both to be passed? If not, the types should be:
```suggestion
  | { fundingUrl: string; sessionToken: never }
  | { sessionToken: string; fundingUrl: never };
```  
@suryatejamandadi-cb

Reply:
believe its best if we allow both to be passed in and default to fundingUrl if both are. thats the existing design